internal/jobqueue/dbjobqueue/dbjobqueue.go

```diff
@@ -657,59 +645,20 @@ func (q *DBJobQueue) JobsUptoByType(jobTypes []string, upto time.Time) (result m
 	return
 }
 
-// Deletes single job and dependencies (recursively)
-func (q *DBJobQueue) DeleteJobIncludingDependencies(jobId uuid.UUID) error {
+func (q *DBJobQueue) DeleteJobResult(jobId uuid.UUID) error {
```

Comment:
What about changing this method to accept a slice of job ids? Nullifying all of them in one query might be faster.

Reply:
True, I just wondered how big the slice could be :|

Reply:
ok I looked it up and it's about 1GB of text, that's a lot of uuids. I'll change this :D